umm-spec-lib/src/cmr/umm_spec/umm_g/project.clj

```diff
@@ -0,0 +1,25 @@
+(ns cmr.umm-spec.umm-g.project
+  "Contains functions for parsing UMM-G JSON projects into umm-lib granule modelProjectRefs
+  and generating UMM-G JSON projects from umm-lib granule model ProjectRefs."
+  (:require
+   [cmr.umm-spec.umm-g.instrument :as instrument]
+   [cmr.umm-spec.util :as util]
+   [cmr.umm.umm-granule :as g])
+  (:import cmr.umm.umm_granule.UmmGranule))
+
+(defn- umm-g-project->ProjectRef
+  "Returns the umm-lib granule model ProjectRef from the given UMM-G Project."
+  [project]
+  (:Campaigns project))
+
+(defn umm-g-projects->ProjectRefs
+  "Returns the umm-lib granule model ProjectRefs from the given UMM-G Projects."
+  [projects]
+  (seq (reduce concat [] (map umm-g-project->ProjectRef projects))))
```

Comment:
Change this line to:
`(seq (distinct (mapcat :Campaigns project)))`, and remove the `umm-g-project->ProjectRef` function.

Reply:
Why are we distincting here?

Reply:
When/if this gets updated to allow for multiple ProjectTypes instead of just a list of short names,  the existing structure is more useful.

Reply:
Are you thinking different campaigns can contain the same short names? So we should distinct here?